AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -1858,7 +1859,7 @@ public static void track(String event, double value, String info) {
    * strings or numbers. You can use up to 200 different parameter names in your app.
    */
 
-  public static void trackGeofence(String event, String info) {
+  public static void trackGeofence(GeofenceEventName event, String info) {
```

Comment:
Thank you @gg4race for the changes. Can you please let me know whats the use of "info" here? Is this any string that the customers can input or this can also be an enum?

Reply:
Not a biggy. Also can you please rename this to "GeofenceEventType", instead of "GeofenceEventTypeName". Sounds like a different type of geofence events.

Reply:
the info would be the name of the geofence that is crossed. These are custom names defined by the customer so probably don't make sense to be enum

Reply:
sounds great.